ompi/mca/coll/libnbc/coll_libnbc_component.c

```diff
@@ -97,7 +98,6 @@ libnbc_open(void)
                               NULL);
     if (OMPI_SUCCESS != ret) return ret;
 
-    OBJ_CONSTRUCT(&mca_coll_libnbc_component.active_requests, opal_list_t);
```

Comment:
Correct, if the free list initialization fails (due to the cache line size bug), then ompi crashes in libnbc_close() as the mca_coll_libnbc_component.active_requests is not initialized. This change justs moves the initialization to an earlier location so that it doesn't crash in libnbc_close().  This kind of misdirected me early on from the original issue.